ansible/playbooks/roles/kubernetes_master/templates/apiserver-logrotate.conf.j2

```diff
@@ -0,0 +1,16 @@
+# {{ansible_managed}}
+
+/var/log/apiserver/*.log {
```

Comment:
How about using path from docs (`/var/log/kubernetes/audit/`)?
Source: https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#log-backend

Reply:
Pointed to that above, we had such path in defaults for a long time, I would not touch that, but can be discussed.

Reply:
This part was removed